exercises/C-comparison-operators/exercise.js

```diff
@@ -7,14 +7,14 @@
 
 var studentCount = 16;
 var mentorCount = 9;
-var moreStudentsThanMentors; // finish this statement
+var moreStudentsThanMentors = true; // finish this statement
```

Comment:
Hi Tanika,
I have done the exact same thing at first, hardcoded the answers.
```suggestion
var moreStudentsThanMentors = studentCount > mentorCount;
```
You can try and do the same for the rest of the file.
Thanks